code/game/machinery/cryopod.dm

```diff
@@ -310,6 +310,31 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		qdel(W)//because we moved all items to preserve away
 		//and yes, this totally deletes their bodyparts one by one, I just couldn't bother
 
+	// Suspend their bank payment
+	if(mob_occupant?.mind?.account_id)
+		var/number_of_bank_account_holder = 0
+		for(var/mob/living/L in GLOB.mob_list)
+			if(L?.mind?.account_id == mob_occupant.mind.account_id)
+				number_of_bank_account_holder++ // if an experimental clone exists, the account will be kept.
+		if(number_of_bank_account_holder <= 1)
+			var/datum/bank_account/target_account = SSeconomy.get_bank_account_by_id(mob_occupant.mind.account_id)
+			target_account.paycheck_bonus = 0
+			target_account.suspended = TRUE // bank account will not be deleted, just suspended
```

Comment:
Make sure that if there is no way to ever access that bank account again you just get rid of it, I don't fancy the idea of having a bunch of useless bank_accounts just floating around.

Reply:
You're right. cards will no longer be allowed to access once it's suspended.